group/templates/group/profile.html

```diff
@@ -35,14 +35,14 @@ <h1>{{ group.name }}</h1>
         {% endfor %}
     </p>
     
-    <div class="d-flex justify-content-center">
+    <div class="d-flex justify-content-center" data-group-id="{{group.id}}">
         {% if user.is_authenticated %}
             {% if user in group.members.all %}
             <a class="btn btn-secondary m-2" href="{% url 'create_event_ingroup' group.id %}">Create Event</a>
-                {% if user is not group.creator %}
+                {% if user not in group.creator %}
```

Comment:
was the previous one wrong?

Reply:
yeah previous was wrong, wasn't working.